include/audio_defines.h

```diff
@@ -20,4 +22,105 @@ enum audio_channel {
 	AUDIO_CH_NUM,
 };
 
+/**
+ * @brief Value to define the data type carried by the block.
+ */
+enum ablk_type {
+	/* The audio data block is raw PCM */
+	ABLK_TYPE_PCM = 1,
+
+	/* The audio data block is coded in LC3 */
+	ABLK_TYPE_LC3
+};
+
+/**
+ * @brief Value to define the PCM interleaving type.
+ */
+enum ablk_interleaved {
+	/* Audio samples are deinterleaved within the buffer */
+	ABLK_DEINTERLEAVED = 1,
+
+	/* Audio samples are interleaved within the buffer */
+	ABLK_INTERLEAVED
+};
+
+/**
+ * @brief A structure giving the description of the audio block
+ */
+struct ablk_block {
+	/* Indicates the data type/encoding of the block. */
+	enum ablk_type data_type;
+
+	/* A pointer to the raw or coded data (e.g., PCM, LC3, etc.) buffer. */
+	void *data;
+
+	/* The size in bytes of the data buffer.
+	 * To get the size of each channel, this value must be divided by the number of
+	 * used channels.
+	 */
+	size_t data_size;
+
+	/* Frame length in microseconds.
+	 * This may be aligned with the Bluetooth connection interval.
+	 * Typical values may be 7500 or 10000 us.
+	 */
+	uint32_t frame_len_us;
```

Comment:
Should we add number_of_frames? If we know how many frames are in the block, we can calculate the bitrate by using data_size and frame_len_us